cmd/generate.go

```diff
@@ -106,6 +106,30 @@ func lookupProjectHcl(m map[string][]string, value string) (key string) {
 	return key
 }
 
+// func stringInSlice(a string, list []string) bool {
+// 	for _, b := range list {
+// 		if b == a {
+// 			return true
+// 		}
+// 	}
+// 	return false
+// }
+
+func sliceUnion(a, b []string) []string {
```

Comment:
nit: add a super brief comment here for what the function does

Reply:
Added